internal/studio/studioHandlers.go

```diff
@@ -502,6 +503,40 @@ func convertWorkflowToComponentsWorkflow(w workflow.Workflow) (components.Workfl
 		return components.Workflow{}, err
 	}
 
+	for key, source := range c.Sources {
+		updatedInputs := lo.Map(source.Inputs, func(input components.Document, _ int) components.Document {
+			// URL
+			if strings.HasPrefix(input.Location, "https://") || strings.HasPrefix(input.Location, "http://") {
+				return input
+			}
+			// Absolute path
+			if strings.HasPrefix(input.Location, "/") {
+				return input
+			}
+			// Registry uri
+			if strings.HasPrefix(input.Location, "registry.speakeasyapi.dev") {
```

Comment:
We have a parser for registry URLs. If possible good to reuse as this could drift when we rename to registry.speakeasy.com